lib/ansible/modules/network/nxos/storage/nxos_devicealias.py

```diff
@@ -0,0 +1,502 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'network'}
+
+
+DOCUMENTATION = '''
+module: nxos_devicealias
+version_added: "2.10"
+short_description: Configuration of device alias.
+description:
+    - Configuration of device alias for Cisco MDS NXOS.
+author:
+    - Suhas Bharadwaj (@srbharadwaj) (subharad@cisco.com)
+options:
+    distribute:
+        description:
+            - Enable/Disable device-alias distribution
+        type: bool
+        default: False
+    mode:
+        description:
+            - Mode of devices-alias, basic or enhanced
+        choices: ['basic', 'enhanced']
+        type: str
+    da:
+        description:
+            - List of device-alias to be added or removed
+        type: list
+        suboptions:
+            name:
+                description:
+                    - Name of the device-alias to be added or removed
+                required:
+                    True
+                type: str
+            pwwn:
+                description:
+                    - pwwn to which the name needs to be associated with
+                type: str
+            remove:
+                description:
+                    - Removes the device-alias if set to True
+                type: bool
+                default: False
+    rename:
+        description:
+            - List of device-alias to be renamed
+        type: list
+        suboptions:
+            old_name:
+                description:
+                    - Old name of the device-alias that needs to be renamed
+                required:
+                    True
+                type: str
+            new_name:
+                description:
+                    - New name of the device-alias
+                required:
+                    True
+                type: str
+'''
```

Comment:
You can add the information of the OS version the module is tested against Refer https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_lldp_global.py#L43

Reply:
Applicable to all the modules in this PR

Reply:
Done